starfish/types/_constants.py

```diff
@@ -21,6 +21,9 @@ class Coordinates(AugmentedEnum):
 
 
 PHYSICAL_COORDINATE_DIMENSION = "physical_coordinate"
+
+LOG = 'log'
+
 """
```

Comment:
This is the documentation for `PHYSICAL_COORDINATE_DIMENSION`.

You need to move the definition for LOG down, and also document it.

Reply:
does it? maybe I should just always store then json encoded info in both the Imagestack and Intensitytable. Since it's just for reading do we care about maintaining the original objects?